core/src/main/java/org/apache/carbondata/core/datamap/DataMapMeta.java

```diff
@@ -19,15 +19,15 @@
 
 import java.util.List;
 
-import org.apache.carbondata.core.indexstore.schema.FilterType;
+import org.apache.carbondata.core.scan.filter.intf.ExpressionType;
 
 public class DataMapMeta {
 
   private List<String> indexedColumns;
 
-  private FilterType optimizedOperation;
+  private List<ExpressionType> optimizedOperation;
```

Comment:
in ExpressionType,    no "like" expression.

Reply:
Currently, the like expression is converted to greater than and less than equal to filter. so there is no LIKE expression in the types.